src/utils/actions.js

```diff
@@ -138,14 +139,15 @@ const actions = [
     translation: "Object.ActionButton.SelectMatchingValues",
     group: "selection",
     getActionCall:
-      ({ app, qStateName, field, value, softLock }) =>
+      ({ app, qStateName, field, value }) =>
       async () => {
         if (field && value) {
           const fieldObj = await app.getField(field, qStateName);
+          const softLock = false;
           await fieldObj.select(value, false, softLock);
```

Comment:
If the softLock arg doesn't do anything, why do we have to pass it? could we just pass two arguments?